pkg/sql/colexec/execgen/inline_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package execgen
+
+import (
+	"testing"
+
+	"github.com/dave/dst"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFormalParamReassignments(t *testing.T) {
+	tests := []struct {
+		funcDecl string
+		callExpr string
+
+		expectedStmts string
+	}{
+		{
+			funcDecl:      `func a() {}`,
+			callExpr:      `a()`,
+			expectedStmts: ``,
+		},
+		{
+			funcDecl:      `func a(a int) {}`,
+			callExpr:      `a(b)`,
+			expectedStmts: `var a int = b`,
+		},
+		{
+			funcDecl: `func a(a int, b int) {}`,
+			callExpr: `a(x, y)`,
+			expectedStmts: `var (
```

Comment:
What's the reason we do `var x <type> = y`, rather than `x := y`? We have a fresh scope right?

Reply:
It's just for cleanliness, I dunno. I like the way a var () decl groups the related assignments together. But it's totally equivalent to n `x := y` statements.

Reply:
True -- no difference.